src/Filter.js

```diff
@@ -38,15 +39,22 @@ class Filter extends Component {
             <button className="gdp-range-button">Find Countries</button>
           </form>
         </div>
-        {/* <div className={this.state.selected === 'Continent' ? 'filter-class' : 'hidden'}>
-          <ul>
-            {this.props.continentData.map((item) => {
-              return <li>{item.name}</li> 
-              })}
-         </ul>
-        </div> */}
+        <div className={this.state.selected === 'Continent' ? 'filter-class' : 'hidden'}>
+          <ul> 
+            {setTimeout(() => {
```

Comment:
I'm curious how you're accessing `this.props.continentData.continents`. It looks like `this.props.continentData` starts off as an array when you're setting it in your App state. Are you changing this into an object as some point? If this were to stay an array, you wouldn't have to do a `setTimeout` because you could always map over an empty array. You also don't want to be changing the data type of values on the fly, it makes your app very prone to errors and bugs.

Reply:
It looks like you are changing this into an object when you get the results of your fetch call back. You would want to only store the continents *array* in that piece of state, not the full object that comes back from your fetch call.